app/views/steps/index.html.erb

```diff
@@ -1,15 +1,36 @@
 <% content_for :header_title, 'Navigation' %>
 <div class="slab">
-  <% StepNavigation.sections.each do |section, steps| %>
-    <h3>
-      <%= section %>
-    </h3>
-    <% steps.each do |step| %>
-      <div>
-        <%= link_to step.to_param.titleize,
-          step_path(step),
-          class: 'button button--small button--full-width' %>
-      </div>
+  <% if snap_application? %>
```

Comment:
since we check for whether there is a snap app or a medicaid app in the controller (and do not allow people to visit this page if there is neither), we *could* make this page a little smarter and have a `steps` method on each class that returns either `StepNavigation` or `MedicaidStepNavigation` and then iterate over that.

so this page could be:

```
current_application.class.step_navigation.sections.each do |section, steps|
  ...
end
```

not perfect, but an option if you'd like to take it

Reply:
ooooooooo interesting. I like it